Applications/MaterialManager/Contracts/Material/Boards/Statistics/BoardTypeInventoryHistory.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+
+using HomagConnect.Base.Contracts.Attributes;
+using HomagConnect.Base.Contracts.Enumerations;
+using HomagConnect.Base.Contracts.Interfaces;
+
+using Newtonsoft.Json;
+
+namespace HomagConnect.MaterialManager.Contracts.Material.Boards.Statistics
```

Comment:
we should change the folder structure to also the term history

Reply:
The Inventory page will be displayed below the Statistics section.